### ISSUE TITLE
Project on sagemaker studio lab not working

### ISSUE THREAD

**Issue by hobbit-frodo** [aws/studio-lab-examples#280]
**Describe the bug**
I've been using a project on 

https://studiolab.sagemaker.aws/

for months in spare time, but since a couple of days ago when I try to start the project I've got the error:

There was a problem when loading your project. This should be resolved shortly. Please try again later.

**To Reproduce**
1. [Go to '...'](https://studiolab.sagemaker.aws/users/MyUser)
3. Start project
4. page says
5. See error

**Expected behavior**
The project is not started as expected and has it has done for weeks

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Linux ubuntu 22.04
 - Browser [e.g. chrome, safari]Fireox 129.0.2 (64-bit)  and also chrome

**Additional context**
![Screenshot_20240823_154509](https://github.com/user-attachments/assets/99b37100-e661-44c2-9dc4-1042cdc01186)

**Comment by Vendetta-S** [comment]
I can voucher that this happens occasionally for me personally, not for weeks on end but occasionally throughout the time I have the account

**Comment by Vendetta-S** [comment]
The solution for this fix is to delete the account and recreate it since it's already approved, the waiting time is short
https://github.com/aws/studio-lab-examples/issues/136
This one said, but it doesn't say anything about the phone number and whether or not it stays, most likely gone.
Deleting the account is a hassle though whenever the error occurs so I'll keep waiting to see if this EVER gets any response...

Edit: As of oct 21, my account seemingly decided to be in this state for literally WEEKS now, Idk how to reach support...

**Comment by MUYIEG** [comment]
The friend above,I have also encountered the same problem. Has it been resolved now?

**Comment by Vendetta-S** [comment]
> The friend above,I have also encountered the same problem. Has it been resolved now?

No, I haven't touched my account in ages and I just gave up, I came back and it still didn't work
I had to use the "delete account and redo it" way and it did approve fast, almost instantly and it removed my phone number so I can add it again
Sadly, you can't fix it any other way
